README.md

```diff
@@ -635,6 +635,43 @@ Translations of the guide are available in the following languages:
    end
    ```
 
+* <a name="parallel-assignment"></a>
+    Avoid the use of parallel assignment for defining variables, except for
+    the return of method calls, use with the splat operator, or variable
+    reassignment. Parallel assignment is less readable and slower.
+<sup>[[link](#parallel-assignment)]</sup>
+
+  ```Ruby
+  # bad
+  a, b, c, d = 'foo', 'bar', 'baz', 'foobar'
+
+  # good
+  a = 'foo'
+  b = 'bar'
+  c = 'baz'
+  d = 'foobar'
+
+  # good - variable reassignment
```

Comment:
Maybe we should go into a bit more details why this particular reassignment is good (as there are other cases which won't be - e.g. `a, b = 1, 2; a, b = 3, 4`.